ui/app/src/Views/AlertList/Parsing.elm

```diff
@@ -4,19 +4,23 @@ import UrlParser exposing ((</>), (<?>), Parser, int, map, oneOf, parseHash, s,
 import Utils.Filter exposing (Filter, parseMatcher, MatchOperator(RegexMatch))
 
 
-boolParam : String -> UrlParser.QueryParser (Maybe Bool -> a) a
-boolParam name =
+parseSilenced : String -> UrlParser.QueryParser (Maybe Bool -> a) a
+parseSilenced name =
```

Comment:
I'm a bit conflicted on this.

On one hand, this is the only place it's being used, so having it be specifically for parsing the silenced param is fine.

On the other hand we had a generic bool parser that we could then just throw a `(parseSilenced >> Maybe.withDefault (Just False)) silencedParam` at (assuming this compiles, haven't tried it :))

Not sure. I'll take either, up to you.